src/main/java/controller/RacingController.java

```diff
@@ -15,12 +16,11 @@ public void run() {
         List<Car> cars = inputView.askCarNames();
         int racingTurn = inputView.askRacingTurn();
 
-        RacingGame racingGame = new RacingGame(cars);
+        RacingGame racingGame = new RacingGame(cars, new RandomMove());
         RacingRecord racingRecord = racingGame.run(racingTurn);
-        racingRecord.saveWinners(racingGame.winnerNames());
 
-        ResultView resultView = new ResultView(racingRecord);
-        resultView.draw();
+        ResultView resultView = new ResultView();
+        resultView.draw(racingRecord);
         resultView.print();
```

Comment:
여전히 view가 상태를 가지고 있네요!
```suggestion
        resultView.print(racingRecord);
```

record를 넘기면 내부적으로 출력할 결과를 만들고 결과를 출력하면 좋을 것 같습니다.

만약에 경기가 이후에 추가로 2개가 열리고
record가 추가될 때 마다 어떤 결과가 나올까요?

---

`resultView.print(racingRecord);` 멱등성을 지켜서 몇번을 출력해도 같은 결과값이 나오는 게 좋습니다.

Reply:
**"view가 상태를 가지고 있다"**가 이해가 되지 않습니다🧐

ResultView에 멤버 필드가 상태라 생각해서 제거하고, 
draw(..) 파라미터로 record를 넘겨서 처리하도록 했습니다만..

Reply:
저의 설명이 부족했나보네요 😭
draw 할 때 마다 멤버변수 sb(상태)에 계속 문자열이 누적되고 있습니다.

Reply:
혹시 StringBuilder 상태 필드를 뜻하는 건가요??

Reply:
넵 맞습니다.

Reply:
알고리즘 문제 풀때 나쁜 습관이 나왔네요 
감사합니다